smarts/core/scenario.py

```diff
@@ -558,7 +559,7 @@ def get_vehicle_start_at_time(
             Start(
                 np.array([pos_x + hhx, pos_y + hhy]),
                 Heading(heading),
-                from_front_bumper=False,
+                from_front_bumper=True,
```

Comment:
This will have a minor affect on traffic history.

Reply:
The `from_front_bumper` was set to `True` as the start position `np.array([pos_x + hhx, pos_y + hhy])` was at the front bumper.